src/main/java/core/basesyntax/operations/PurchaseOperation.java

```diff
@@ -4,9 +4,10 @@
 
 public class PurchaseOperation implements OperationHandler {
     @Override
-    public void updateDB(String fruit, int quantity, Storage storage) {
+    public void handle(String fruit, int quantity, Storage storage) {
         if (storage.getStorage().get(fruit) - quantity < 0) {
-            throw new RuntimeException("Quantity value can't be less than zero");
+            throw new RuntimeException("Current fruit quantity is "
+                    + storage.getStorage().get(fruit));
         }
         storage.getStorage().put(fruit, storage.getStorage().get(fruit) - quantity);
```

Comment:
You are using `storage.getStorage().get(fruit)` several times here, maybe it worths creating variable for better readability?